install/assets/defaults/30-freescout

```diff
@@ -1,5 +1,6 @@
 #!/command/with-contenv bash
 
+DB_TYPE=${DB_PORT:-"mysql"}
```

Comment:
Isn't this supposed to be `DB_TYPE=${DB_TYPE:-"mysql"}`?

Reply:
Ack - yes. Fixing.

Reply:
Fixed with `1.16.2`